### ISSUE TITLE
Have vocal isolation be optional.

### ISSUE THREAD

**Issue by AlextwoWasTaken** [SociallyIneptWeeb/AICoverGen#98]
This program is great, but sometimes the isolation is a problem for me, I want to make an ai cover for "I got no time", I have the official vocals but because it does the isolation it thinks most of the vocals are part of the instrumental. I've looked everywhere and couldn't find anything to turn this off. This would really help if it were implemented!

**Comment by xezaf14** [comment]
Yes, the author should make that a priority, knowing that I can always do a better instrumental version than the software, it would be great to just import it and run.
